app/modules/new-vm/index.coffee

```diff
@@ -173,6 +174,11 @@ module.exports = angular.module 'xoWebApp.newVm', [
         VDI.id = VDI_id++
         VDI.size = bytesToSizeFilter VDI.size
         VDI.SR or= default_SR
+      # if the template is labeled CoreOS
+      # we'll use config drive setup
+      if template.name_label == 'CoreOS'
+        xoApi.call('vm.getCloudInitTemplate', template.id).then (config) ->
```

Comment:
@julien-f : I got `Unhandled rejection Error: no such deferred` when using this call. Any idea?